Source/swiftlint/Commands/Baseline.swift

```diff
@@ -0,0 +1,89 @@
+import ArgumentParser
+import Foundation
+import SwiftLintFramework
+
+extension SwiftLint {
+    struct Baseline: ParsableCommand {
+        static let configuration = CommandConfiguration(
+            abstract: "Reports the violations in a baseline or the violations " +
+                      "that are present in another baseline, but not in the original.",
```

Comment:
The existing abstracts for other commands are quite short (and come without a final dot). For a the sake of consistency, let's do it likewise here:
```suggestion
            abstract: "Operations on existing baselines",
```